gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -798,6 +804,24 @@ gst_tensor_transform_set_property (GObject * object, guint prop_id,
       filter->acceleration = FALSE;
 #endif
       break;
+    case PROP_APPLY:
+    {
+      gint64 val;
+      const gchar *param = g_value_get_string (value);
+      gchar **strv = g_strsplit_set (param, ",", -1);
+      gint i, num = g_strv_length (strv);
+
+      for (i = 0; i < num; i++) {
+        val = g_ascii_strtoll (strv[i], NULL, 10);
+        if (errno == ERANGE) {
+          ml_loge ("Overflow occured during converting %s to a gint64 value",
+              strv[i]);
+        }
+        filter->apply = g_list_append (filter->apply, GINT_TO_POINTER (val));
```

Comment:
filter->apply is leak, plz add free function in finalize function.

Reply:
Updated.